functional_tests/scylla_operator/test_functional.py

```diff
@@ -75,75 +72,82 @@ def test_single_operator_image_tag_is_everywhere(db_cluster):
 
     # NOTE: operator's image is used in many places. So, walk through all of the related namespaces
     for namespace in (SCYLLA_NAMESPACE, SCYLLA_MANAGER_NAMESPACE, SCYLLA_OPERATOR_NAMESPACE):
-        pods = yaml.safe_load(db_cluster.k8s_cluster.kubectl(
-            "get pods -o yaml", namespace=namespace).stdout)["items"]
+        pods = yaml.safe_load(db_cluster.k8s_cluster.kubectl("get pods -o yaml", namespace=namespace).stdout)["items"]
         for pod in pods:
             for container_type in ("c", "initC"):
                 for container in pod.get("status", {}).get(f"{container_type}ontainerStatuses", []):
                     image = container["image"].split("/")[-1]
                     if image.startswith("scylla-operator:") and image.split(":")[-1] != expected_operator_tag:
-                        pods_with_wrong_image_tags.append({
-                            "namespace": namespace,
-                            "pod_name": pod["metadata"]["name"],
-                            "container_name": container["name"],
-                            "image": image,
-                        })
+                        pods_with_wrong_image_tags.append(
+                            {
```

Comment:
I find it counterproductive to use more new lines than it is really needed like here.
If we have bunch of single-element parent wrappers it doesn't mean it is useful to use new line for entering and closing of each of them.

Reply:
Thanks for the links.